### PR TITLE
2 packages from ocsigen/lwt at 5.9.0

### PR DESCRIPTION
**Pull request by raphael-proust**
This pull-request concerns:
- `lwt.5.9.0`: Promises and event-driven I/O
- `lwt_retry.5.9.0`: Utilities for retrying Lwt computations

---
* Homepage: https://github.com/ocsigen/lwt
* Source repo: git+https://github.com/ocsigen/lwt.git
* Bug tracker: https://github.com/ocsigen/lwt/issues

---
:camel: Pull-request generated by opam-publish v2.4.0

### CODE REVIEW

**Review by shonfeder — APPROVED** [review]

**Comment by shonfeder** [comment]
Looks great. Thank you for cutting the releasth

File: packages/lwt_retry/lwt_retry.5.9.0/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+maintainer: [
+  "Raphaël Proust <code@bnwr.net>" "Shon Feder <shon.feder@gmail.com>"
+]
+authors: ["Shon Feder"]
+license: "MIT"
+homepage: "https://github.com/ocsigen/lwt"
+doc: "https://ocsigen.org/lwt"
+bug-reports: "https://github.com/ocsigen/lwt/issues"
+depends: [
+  "dune" {>= "2.0"}
```
Comment (mseri) [inline packages/lwt_retry/lwt_retry.5.9.0/opam:13]:
```suggestion
  "dune" {>= "2.7"}
```
for
```

#=== ERROR while compiling lwt_retry.5.9.0 ====================================#
# context              2.4.0~alpha1~dev | linux/x86_64 | ocaml-base-compiler.4.11.2 | pinned(https://github.com/ocsigen/lwt/archive/refs/tags/5.9.0.tar.gz)
# path                 ~/.opam/4.11/.opam-switch/build/lwt_retry.5.9.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p lwt_retry -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/lwt_retry-8-e30a62.env
# output-file          ~/.opam/log/lwt_retry-8-e30a62.out
### output ###
# File "dune-project", line 1, characters 11-14:
# 1 | (lang dune 2.7)
#                ^^^
# Error: Version 2.7 of dune is not supported.
# Supported versions:
# - 1.0 to 1.12
# - 2.0
```

File: packages/lwt_retry/lwt_retry.5.9.0/opam [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "ocaml" {>= "4.08"}
+  "lwt" {>= "5.3.0"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (mseri) [inline packages/lwt_retry/lwt_retry.5.9.0/opam:18]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by mseri** [comment]
The tests were not shouwing suspicious failures I am going to merge as soon as the lint is green. @raphael-proust please update the bounds for dune (this should also solve the dev <-> pinned issue when generating the opam file)

**Comment by raphael-proust** [comment]
I've fixed things upstream

Thanks a lot for making these fixes which I didn't have a lot of time to dedicate to

<3

**Pull request merged**